packages/story-editor/src/components/panels/design/layer/group.js

```diff
@@ -0,0 +1,512 @@
+/*
```

Comment:
Woah, a brand new component with 512 lines? I assume this has a ton of overlaps with the regular layer component, but it's left as-is because we already have #11576 to deal with it?